.github/workflows/ports_renesas-ra.yml

```diff
@@ -0,0 +1,24 @@
+name: renesas-ra port
+
+on:
+  push:
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+      - 'tools/**'
+      - 'py/**'
+      - 'extmod/**'
+      - 'lib/**'
+      - 'drivers/**'
+      - 'ports/ra/**'
```

Comment:
this should be `/ports/renesas-ra/**`

Reply:
Thank you for pointing that out. I have updated.